src/components/InputField/InputField.tsx

```diff
@@ -5,12 +5,14 @@ type InputFieldProps = {
   label: string;
   fieldType: "POSTCODE" | "AMOUNT" | "PERCENTAGE" | "TEXT";
   required: boolean;
+  formatting: boolean;
 };
 
 const InputField = ({
   label,
   fieldType,
   required,
+  formatting,
```

Comment:
This should not be a prop, and instead just used the usual $ sign on AMOUNT fieldtypes.